packages/yew/src/lib.rs

```diff
@@ -1,6 +1,7 @@
 #![allow(clippy::needless_doctest_main)]
 #![doc(html_logo_url = "https://yew.rs/img/logo.png")]
 #![cfg_attr(documenting, feature(doc_cfg))]
+#![doc(html_playground_url = "https://play.yew.rs/")]
```

Comment:
Should this somehow include the version of yew in use to get dependencies right? Currently, only one version is really relevant, but early cracks are showing: clicking "Run" on the docs for yew master opens the playground compiling with yew-0.19.3.

Reply:
The playground only supports 0.19 at the moment. It shouldn't be too hard to add support for master as well. We would be running another app-compiler but the app is based on master. It would likely be nightly builds for Yew (only if changes are there)

Do you think we should just remove it for now?

Reply:
Yeah let's not include it in the docs for now, the links on yew.rs are helpful already.

Reply:
```suggestion
```